### PR TITLE
OCP 11 certification

### PR DESCRIPTION
**Pull request by kthoms**

### CODE REVIEW

File: _posts/2021-04-25-ocp-java11.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+categories: [Java, Certification]
+header:
+  text: OCP 1Z0-819 Experience Report
+  image: post
+---
+
+On April 23th 2021 I have done the OCP Java 11 Developer (1Z0-819) exam and passed. Oracle released this new certification
+in October 2020 as a successor of Java SE 11 Programmer I (1Z0-815) and Java SE 11 Programmer II (1Z0-816) combined in one.
+
+It was an intensive experience, and I wanted to share something about the motivation, preparation and the exam itself.
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:19]: ... to share ~~something~~ my thoughts and insights about...

File: _posts/2021-04-25-ocp-java11.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+The views expressed here are subjective. I try to give you a detailed insight why and how I started to prepare for the exam
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:20]: ...detailed insight _**into** or **regarding**_ why...

File: _posts/2021-04-25-ocp-java11.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+to the point where I performed the exam. Hopefully you will find some inspiration and tips for your own journey
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:21]: Comma after Hopefully.

File: _posts/2021-04-25-ocp-java11.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+to get the certification.
+
+## Why doing the Java Certification?
+
+I'm now working with Java since my study days 25 years ago and got first in touch with the language in its 1.0 version, mainly developing small applets
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:26]: This paragraph is a bit hard to follow. Consider rephrasing a bit.

File: _posts/2021-04-25-ocp-java11.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+
+On April 23th 2021 I have done the OCP Java 11 Developer (1Z0-819) exam and passed. Oracle released this new certification
+in October 2020 as a successor of Java SE 11 Programmer I (1Z0-815) and Java SE 11 Programmer II (1Z0-816) combined in one.
+
+It was an intensive experience, and I wanted to share something about the motivation, preparation and the exam itself.
+The views expressed here are subjective. I try to give you a detailed insight why and how I started to prepare for the exam
+to the point where I performed the exam. Hopefully you will find some inspiration and tips for your own journey
+to get the certification.
+
+## Why doing the Java Certification?
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:24]: do~~ing~~

File: _posts/2021-04-25-ocp-java11.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+I'm now working with Java since my study days 25 years ago and got first in touch with the language in its 1.0 version, mainly developing small applets 
+and for studying computer languages. From my first day as a consultant back in 2000 I am programming with Java almost every day. Languages, language workbenches 
+and developing tools for developers have become a main part of my work, so my Java skills are solid.
+
+I never felt that it is necessary to have a Java certification in order to accomplish to work on complex Java based systems. I believe even for potential customers
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:30]: ...in order to ~~accomplish to work~~ **successfully work** on complex Java-based systems.

File: _posts/2021-04-25-ocp-java11.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+to get the certification.
+
+## Why doing the Java Certification?
+
+I'm now working with Java since my study days 25 years ago and got first in touch with the language in its 1.0 version, mainly developing small applets 
+and for studying computer languages. From my first day as a consultant back in 2000 I am programming with Java almost every day. Languages, language workbenches 
+and developing tools for developers have become a main part of my work, so my Java skills are solid.
+
+I never felt that it is necessary to have a Java certification in order to accomplish to work on complex Java based systems. I believe even for potential customers 
+who order profiles of consultants it is obvious from reading mine that I have a rock solid project experience and
```
Comment (grigala) [inline _posts/2021-04-25-ocp-java11.md:31]: This is a bit fuzzy too. I'd say something along those lines:

> I believe that even for potential customers, that are reviewing applicant profiles, it is immediately obvious that my experience is rock-solid. Whether I have a certificate or not, does not really matter.

**Review by grigala — CHANGES_REQUESTED** [review]
I'll come back to it tomorrow.

File: _posts/2021-04-25-ocp-java11.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+
+At Karakun we have an education budget that we can use on our behalf, but to be honest,
+preparing such an exam also involves a high degree of spare time. We are also quite busy with our daily projects, and to take time it is not 
+always the right time in a project.
```
Comment (georgebax) [inline _posts/2021-04-25-ocp-java11.md:56]: > to take time it is not always the right time in a project
This is not very understandable. Did you mean "taking time is not always possible" ?

File: _posts/2021-04-25-ocp-java11.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+
+As it comes, at that time I was deeply busy with my current main customer project. It afforded almost all of my time, and there was almost no room to
+got started. Fast I was lacking behind the schedule and thought it is not possible to work through the complete book in that time. But then came easter
```
Comment (georgebax) [inline _posts/2021-04-25-ocp-java11.md:59]: > got started
```suggestion
get started
```

File: _posts/2021-04-25-ocp-java11.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+vacation and "thanks" to the corona crises and bad weather there was anyway time left then. Thus I decided to work through the chapters almost each vacation
```
Comment (georgebax) [inline _posts/2021-04-25-ocp-java11.md:60]: cris~~e~~is

File: _posts/2021-04-25-ocp-java11.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+
+When I got the call from the proctorer she asked me to take the notebook and show around my desk. I had to remove some more stuff from the desk, and finally
+the exam was released.
+
+## The Exam
+
+Now it was finally time. All or nothing! When I started the test the counter began. 90 minutes for 50 questions.
+
+The questions were completely distributed over the topics that are covered by the Study Guide. From datatypes, loops, lambdas, modular applications to
+security and JDBC. The mock exams simulate this quite well, the questions are distributed similar.
```
Comment (georgebax) [inline _posts/2021-04-25-ocp-java11.md:178]: similar**ly**

File: _posts/2021-04-25-ocp-java11.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+
+I opened a link and it got some information. One of them was that no external monitors are allowed. I wanted to avoid problems and unplugged mine, switching
+to the notebook's display and camera. If I would have read the tips carefully then there you read that an external monitor is actually allowed
+when the notebook is closed. I was just confused by the message on the web page. Then the check-in software requested to make a photo of yourself, then front and back of your identity document and photos of your room in front of desk, 
+back, left and right.
+
+Finally I was done there and got into the waiting queue. There was a counter how many people are checked in before you. I got about position 25, and it
+took between 5 and 10 minutes until I was in front.
+
+Now a proctorer opened a chat window and wrote that he could not see my video stream. This will be surely because I unplugged the external monitor and
```
Comment (georgebax) [inline _posts/2021-04-25-ocp-java11.md:161]: I would use proctor instead of proctorer (I can't see if this word exists).

File: _posts/2021-04-25-ocp-java11.md [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+combined in one. In this article you will find a personal experience report from taking the online
+exam, starting from preparation over study to the exam day.
+"
+categories: [Java, Certification]
+header:
+  text: OCP 1Z0-819 Experience Report
+  image: post
+---
+
+On April 23th 2021 I have done the OCP Java 11 Developer (1Z0-819) exam and passed. Oracle released this new certification
```
[the remark this answers is not in the document]
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:16]: ok, sounds better

File: _posts/2021-04-25-ocp-java11.md [hunk 14]
```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title:  'Experience Report: Taking the Online OCP Java SE 11 Developer Certification (1Z0-819)'
+author: karsten
+featuredImage: java11-omg
+excerpt: "OCP Java SE 11 Developer (1Z0-819) is the successor of previous OCA and OCP certifications
```
[the remark this answers is not in the document]
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:6]: changed

File: _posts/2021-04-25-ocp-java11.md (hunk 1 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:19]: ok

File: _posts/2021-04-25-ocp-java11.md (hunk 2 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:20]: chose _into_

File: _posts/2021-04-25-ocp-java11.md (hunk 3 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:21]: not sure, but I have added it. The rule of thumb I learned in English class was "If in doubt leave it out"

File: _posts/2021-04-25-ocp-java11.md (hunk 7 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:31]: sounds better, thanks!

File: _posts/2021-04-25-ocp-java11.md (hunk 8 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:56]: yes, meant that. rephrased the paragraph.

File: _posts/2021-04-25-ocp-java11.md (hunk 12 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:161]: oh right ;)

File: _posts/2021-04-25-ocp-java11.md (hunk 4 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:26]: I removed the last sentence, is it better now?

File: _posts/2021-04-25-ocp-java11.md [hunk 15]
```diff
@@ -0,0 +1,290 @@
...
+After I finished the book I went through all wrongly answered questions again.
+I still failed with some of them, but the hit rate was much better.
+
+Overall the Java SE 11 Developer Complete Study Guide is very well written.
+It describes everything in detail, one topic after the other without overlapping or taking anything for granted.
+Also, the code examples are well chosen and helpful for understanding.
+The writing style was very easy to follow.
+I really can recommend reading the book!
+
+I learned really many details where I do not think about anymore.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:107]:
```suggestion
While studying I learned many details of Java about which I do not worry in my daily work.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 16]
```diff
@@ -0,0 +1,290 @@
...
+The compiler, the IDE, Dr. Google or Mr. Stackoverflow helps me to correct things when I'm doing something wrong, and fixing is often easy.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:108]:
```suggestion
The compiler, the IDE, Dr. Google or Mr. Stackoverflow help to correct me when I'm doing something wrong.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 17]
```diff
@@ -0,0 +1,290 @@
...
+The book helped to focus on those details and think in front.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:109]:
```suggestion
The book helped to focus on those details and think ahead.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 18]
```diff
@@ -0,0 +1,290 @@
...
+
+## About certain exam topics
+
+You'll need to learn many details of the language and its tools.
+Some of them you'll likely never need.
+Some you even probably should not use.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:115]:
```suggestion
Some you even probably should not use.
The book also tries to set these details into the context of modern day programming.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 19]
```diff
@@ -0,0 +1,290 @@
...
+You'll see that with Java you could write code that is more complicated to understand than Perl scripts written by a drunk gorilla.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:116]:
```suggestion
You'll also see that it is possible to write code in Java that is more complicated to understand than Perl scripts written by a drunk gorilla.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 20]
```diff
@@ -0,0 +1,290 @@
...
+
+Especially tricky were to me nested loops with incrementing different scoped variables with either pre- or post-increment operator and proper usage of `break` or `continue` on labeled loops.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:118]:
```suggestion
For me, especially hard to understand and follow are nested loops with incrementing different scoped variables using either pre- or post-increment operator and usage of `break` or `continue` in conjunction with labeled loops.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 21]
```diff
@@ -0,0 +1,290 @@
...
+To answer those questions correctly you'll have to track the variable state.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:119]:
```suggestion
To answer those questions correctly you'll have to track the variable state step by step through the execution of the program.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 22]
```diff
@@ -0,0 +1,290 @@
...
+Although you really should understand how and when to use `break` and `continue`, doing this together with complicated incremented variable state just makes knots in your brain.
+During study I took the time to follow the program flow and state, but for the exam I decided to just skip such a question.
+It just takes too much time, and one question is as much worth as the other.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:122]:
```suggestion
It simply takes too much time, and one question is worth as many points as any other.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 23]
```diff
@@ -0,0 +1,290 @@
...
+
+For the exam you'll learn all the command line switches and their inconsistencies when they use single or double hyphen.
+Also options for commands like `jmod`.
+Actually I did not know that command before.
+So my taking was here that this command exists and roughly its purpose.
+But why you need to memorize which command line options it offers and get tricked by options that might look right?
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:128]:
```suggestion
But why do you need to memorize which command line options it offers and get tricked by options that might look right?
```

File: _posts/2021-04-25-ocp-java11.md [hunk 24]
```diff
@@ -0,0 +1,290 @@
...
+Actually more then I would like to.
+Nowadays I don't really care much about Java object serialization.
+The basic rules are well understood.
+It was quite interesting to learn the order of object initialization, i.e. which constructor is called or not and in which order.
+I never really cared so far.
+I read about options to influence what and how state is (de-) serialized which I'll never use again.
+However, now I know there are more options available than I was aware of before.
+
+Of course there were also topics which I use daily, but where the book really helped me to learn and think about in detail. I especially enjoyed here
+the chapters about Generics, Functional Programming, Modular Applications, Concurrency and NIO.2. This really helped me to improve my hard skills.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:143]:
```suggestion
Of course there were also topics which I use daily, but where the book really helped me to learn and think about in detail.
I especially enjoyed here the chapters about Generics, Functional Programming, Modular Applications, Concurrency and NIO.2.
This really helped me to improve my hard skills.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 25]
```diff
@@ -0,0 +1,290 @@
...
+
+## Mock Exams
+
+Additionally to the Study Guide I ordered the [1Z0-819-J Mock Exams from Enthuware](https://enthuware.com/java-certification-mock-exams/oracle-certified-professional/ocp-java-11-exam-1Z0-819-J).
+I read about them when I was looking into some threads about the certification in the [OCP Forum at Coderanch](https://coderanch.com/f/24/java-programmer-OCPJP).
+I was at this forum to report some small possible errata in the book.
+In the threads some developers were mentioning the mock exams and I thought it could be worth to try some additionally to the review questions in the book.
+
+Before the exam I just had to do 7 of those mock exams (there are 22 with ~50 questions each).
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:152]:
```suggestion
Before the exam I only had time to do 7 of those mock exams (there are more than 20 with ~50 questions each).
```

File: _posts/2021-04-25-ocp-java11.md [hunk 26]
```diff
@@ -0,0 +1,290 @@
...
+After doing this I can really recommend to use them when preparing for the exam.
+While it is possible here to get offered how many answers you actually have to choose, the questions themselves felt partially even trickier than in the book.
+
+You can either review them after finishing the complete test or choose to be able to review after each question.
+I started with the first option and reviewed the questions where I failed.
+The different options are commented and help to understand the topic in more detail.
+For the last mock exams I decided to use the option to review after each question, and used this to understand also for correct answers if they were answered correctly because I really hit the nail there.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:159]:
```suggestion
For the last mock exams I decided to use the option to review after every question, and used this to understand also for correct answers if they were answered correctly because I really understood the topic.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 27]
```diff
@@ -0,0 +1,290 @@
...
+Sometimes you might be just randomly right, and I avoided false positives.
+Of course, this takes time then.
+
+Overall the mock exams do quite well simulate the real exam in their content and extend.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:163]:
```suggestion
Overall the mock exams do quite well simulating the real exam in their content and extend.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 28]
```diff
@@ -0,0 +1,290 @@
...
+
+I started the login procedure as recommended 30 minutes before planned start.
+You'll need to start the OnVUE software and you'll get a code from the login link in the email you received.
+Before I mostly cleaned my desk except for the external monitor I wanted to work with my Notebook.
+You need to set the camera you are using at first.
+For me, I had the notebook closed and chose the external webcam.
+The software showed me then the camera stream, so it looked fine so far.
+
+The next step involved using the smartphone for the check-in procedure.
+You have 2 options to continue: Either get an SMS with a link, or enter the link yourself.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:186]:
```suggestion
You have 2 options to continue: Either get an text message with a link, or enter the link yourself.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 29]
```diff
@@ -0,0 +1,290 @@
...
+I chose the SMS, but unfortunately I did not receive the message.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:187]:
```suggestion
I chose the text message, but unfortunately I did not receive the message.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 30]
```diff
@@ -0,0 +1,290 @@
...
+The OnVUE software was waiting until you continue on the smartphone and I was blocked.
+I needed to restart and the software and then entered the link manually.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:189]:
```suggestion
I needed to restart the software and then entered the link manually.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 31]
```diff
@@ -0,0 +1,290 @@
...
+I opened a link and it got some information.
+One of them was that no external monitors are allowed.
+I wanted to avoid problems and unplugged mine, switching to the notebook's display and camera.
+If I would have read the tips carefully then there you read that an external monitor is actually allowed when the notebook is closed.
+I was just confused by the message on the web page.
+Then the check-in software requested to make a photo of yourself, then front and back of your identity document and photos of your room in front of desk, back, left and right.
+
+Finally I was done there and got into the waiting queue.
+There was a counter how many people are checked in before you.
+I got about position 25, and it took between 5 and 10 minutes until I was in front.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:201]:
```suggestion
I got about position 25, and it took between 5 and 10 minutes until I was first in line.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 32]
```diff
@@ -0,0 +1,290 @@
...
+
+Now a proctor opened a chat window and wrote that he could not see my video stream.
+This will be surely because I unplugged the external monitor and the webcam during the check-in procedure.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:204]:
```suggestion
This must have been because I unplugged the external monitor and the webcam during the check-in procedure.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 33]
```diff
@@ -0,0 +1,290 @@
...
+However, since the software still showed me in a video stream, I was surprised that the proctor did not see it, too.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:205]:
```suggestion
However, since the software still showed me in a video stream, I was surprised that the proctor did not see it as well.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 34]
```diff
@@ -0,0 +1,290 @@
...
+So I needed to restart the software again and go through the check-in procedure again.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:206]:
```suggestion
So I needed to restart the software once more and go through the entire check-in procedure again.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 35]
```diff
@@ -0,0 +1,290 @@
...
+It was already close to the planned starting time.
+The check-in involved again opening the link, fortunately this time I did not need to take the photos again.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:208]:
```suggestion
The check-in involved opening the link again, fortunately this time I did not need to take the photos.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 36]
```diff
@@ -0,0 +1,290 @@
...
+
+The questions were completely distributed over the topics that are covered by the Study Guide.
+From datatypes, loops, lambdas, modular applications to security and JDBC.
+The mock exams simulate this quite well, the questions are distributed similarly.
+
+You can skip questions that you will answer later.
+When completing the last question you have the option to answer all unanswered question.
+One tip I can give is: Do this.
+Decide fast if a question will take you long to understand.
+This can be either by more code, or more complex code.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:233]:
```suggestion
This can be either by due to a large number of lines of code, or very complex code structure.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 37]
```diff
@@ -0,0 +1,290 @@
...
+Since each question weights as much as others, just risk that you'll not answer those and get points on others.
+You'll need to decide this fast!
+Of course, you should do this only for few questions.
+I had about 4. Already planned for skipping was nested labeled loops where you need to answer where to insert `break <LABEL>` or `continue <LABEL>` to get a specific output.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:237]:
```suggestion
I had about 4.
Already planned for skipping was nested labeled loops where you need to answer where to insert `break <LABEL>` or `continue <LABEL>` to get a specific output.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 38]
```diff
@@ -0,0 +1,290 @@
...
+The time is well spent on others.
+
+Overall the questions were not easy and often tricky, but you know that from practicing the questions in the Study Guide and the mock exams.
+I had the feeling that the mock exam questions were a bit harder than the questions I had in the exam.
+So if you are doing well on those tests, you are well prepared!
+
+There were some questions of course that cause compilation errors.
+As recommended follow the call sequence and ask yourself "Does it compile?".
+I spotted some errors like datatype conversion, extending/overriding final classes / methods, and reducing visibility of overridden methods.
+Could be that I missed some, at the end I made of course some mistakes.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:248]:
```suggestion
It is likely that I missed some, at the end I did not a 100% score.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 39]
```diff
@@ -0,0 +1,290 @@
...
+I had a moderate paste in going through the questions.
+Each question I read with care and also the answers.
+After doing this I was often clear which options I would answer.
+Sometimes I was not completely sure and marked those questions for review later when I was done with all.
+
+At the end I had less about 10 minutes for the skipped and marked questions.
+I needed that time to answer the skipped ones, and no time was left to review unsure ones.
+For the mentioned nested loop question I just did a wild guess and it surely was wrong.
+
+After the time was over the exam finished automatically and you get an immediate answer if you passed and the correct answer percentage.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:260]:
```suggestion
After the time was up the exam finished automatically and you get the result immediately if you passed and the correct answer percentage.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 40]
```diff
@@ -0,0 +1,290 @@
...
+I was glad that I passed and this was all over now.
+
+# Conclusion
+
+Doing this exam is an intense experience and requires well preparation, even for experienced Java developers.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:265]:
```suggestion
Doing this exam is an intense experience and requires thorough preparation, even for experienced Java developers.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 41]
```diff
@@ -0,0 +1,290 @@
...
+When you are experienced take this as a chance to learn details and challenge yourself.
+Hopefully you are not doing the exam because you need just the badge in your CV and just want to pass somehow, but really learn and improve your skills.
+
+I recommend studying the OCP Java 11 Complete Study Guide in detail, and additionally practicing with the Mock Exams from Enthuware.
+If you do this you are really well prepared for taking the exam!
+However, do not underestimate the effort for doing so.
+Wouldn't I have sacrificed some of the vacation and spare time, I would not have made it through the study material.
+I might have passed, but when I would have before studying the whole material, I won't have touched them again.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:273]:
```suggestion
If I wouldn't had sacrificed a large part of my vacation, and even more spare time, I would never made it through the material in time to do the exam before the 25th.
I might have passed, but when I would have before studying the whole material, I wouldn't have come back simply to read the rest.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 42]
```diff
@@ -0,0 +1,290 @@
...
+Not all of our study group made it through the material because they did not have enough time besides their regular work.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:274]:
```suggestion
Not everyone in the study group made it through the material in time.
Mainly because they did not have enough time besides their regular work.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 43]
```diff
@@ -0,0 +1,290 @@
...
+They decided to just make the exam later, then of course at regular cost, when they learned completely for it.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:275]:
```suggestion
They decided to take the exam later once they have finished learning, paying the regular cost.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 44]
```diff
@@ -0,0 +1,290 @@
...
+Just trying to pass was not the right thing for them.
+
+On exam day make sure that you check in early and have if possible nothing then your computer on the desk and within arm reach.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:278]:
```suggestion
On exam day make sure that you check in early and have nothing beside your computer on the desk and within arms reach.
```

File: _posts/2021-04-25-ocp-java11.md [hunk 45]
```diff
@@ -0,0 +1,290 @@
...
+
+Take enough time to read the questions and their answers carefully.
+Follow the program flow from the main method to what is instantiated and called.
+
+Skip questions immediately when they'll cost you valuable time reading and understanding them.
```
Comment (sclassen) [inline _posts/2021-04-25-ocp-java11.md:283]:
```suggestion
Skip questions immediately when they will cost you valuable time reading and understanding them.
```

**Review by sclassen — COMMENTED** [review]
Nicht erschreken an der grossen Zahl von Vorschlägen.
Ich mag den Text sehr und habe oft nur kleine Worte oder die Satzstellung angepasst.

**Comment by kthoms** [comment]
Ganz lieben Dank für das Feedback @sclassen !

File: _posts/2021-04-25-ocp-java11.md (hunk 27 above)
Comment (kthoms) [reply _posts/2021-04-25-ocp-java11.md:163]: i think 'simulate' is correct here
